ServiceDesk.Abstractions/DTOs/ServiceDeskRequestDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace ServiceDesk.Abstractions.DTOs
+{
+    public class ServiceDeskRequestDTO
+    {
+        public string RequestNumber { get; set; }
+
+        public DateTime Created { get; set; }
+
+        public string Title { get; set; }
+
+        public string StatusName { get; set; }
+
+        public string ExecutorFullName { get; set; }
+
+        public string ExecutorIconUrl { get; set; }
```

Comment:
This field is not needed. And keep the previous one, we'll use it in bot dialog.

Reply:
Prop has been removed